rclcpp/test/test_publisher_subscription_count_api.cpp

```diff
@@ -51,6 +51,10 @@ class TestPublisherSubscriptionCount : public ::testing::TestWithParam<TestParam
   {
     rclcpp::init(0, nullptr);
   }
+  static void TearDownTestCase()
+  {
+    rclcpp::shutdown();
+  }
```

Comment:
Please, lets resolve this in the other pull request:

https://github.com/ros2/rclcpp/pull/1073

And remove these from this unrelated pull request.

Reply:
Will do, sorry for the local changes.